test/unit/deadline_client/cli/test_cli_attachment.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Tests for the CLI attachment commands.
+"""
+import os
+
+from click.testing import CliRunner
+
+from deadline.client.cli import main
+
+
+def test_cli_attachment_existence(fresh_deadline_config):
+    """
+    Confirm that the CLI is not availble for environment with no JOB_ATTACHMENT_CLI variable
+    """
+
+    assert not os.environ.get("JOB_ATTACHMENT_CLI")
```

Comment:
See in my PR how I was able to test this!

I did a `main.add_command` in the test, so tests always supports the new CLI even without the env variable.